source/features/avoid-accidental-submissions.tsx

```diff
@@ -0,0 +1,39 @@
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function addQuickSubmit(): void {
+	select('input#commit-summary-input')!.classList.add('js-quick-submit');
+}
```

Comment:
Not sure why we added this here. Doesn't this **allow** submission via ctrl-enter? It should be extracted to its own feature (and the shortcut should mention `cmd` on Mac )